src/components/Card/Card.tsx

```diff
@@ -7,19 +7,19 @@ interface CardDetails {
 
 import "./Card.css"
 
-function StayCard(props: CardDetails) {
+function Card({ImageWebp, ImageJpg}: CardDetails) {
```

Comment:
ideally just have an image props: which has all these variety of fields as keys within it